Automattic/jetpack
# Add connection button to Welcome banner

Fixes pbNhbs-9bI-p2#comment-19686

<img width="1698" alt="image" src="https://github.com/Automattic/jetpack/assets/5714212/232222df-2d37-4383-883d-9d425d0b29a7">


## Proposed changes:

* This PR adds a button to the welcome banner in My Jetpack that links to the connection screen in case a user is not connected yet

### Other information:


## Jetpack product discussion



pbNhbs-9bI-p2#comment-19686

## Does this pull request change what data or activity we track or use?


Just another Tracks event

## Testing instructions:






* Spin up a JN site with Jetpack Beta pointing to this branch
* Without connecting to Jetpack, go to My Jetpack
* Confirm you see the Welcome banner, and that there's a button there
* When clicked, this button should trigger recordEvent (you can filter by the event name in the network tab of your browser's console), and redirect to the connection screen
